src/sentry/api/endpoints/event_attachment_details.py

```diff
@@ -8,12 +8,43 @@
 except ImportError:
     from django.http import StreamingHttpResponse
 
-from sentry import features, options
-from sentry.api.bases.project import ProjectEndpoint
-from sentry.models import Event, SnubaEvent, EventAttachment
+from sentry import features, options, roles
+from sentry.api.bases.project import ProjectEndpoint, ProjectPermission
+from sentry.models import Event, SnubaEvent, EventAttachment, OrganizationMember
+from sentry.api.serializers.models.organization import ATTACHMENTS_ROLE_DEFAULT
+
+
+class EventAttachmentDetailsPermission(ProjectPermission):
+    def has_object_permission(self, request, view, project):
+        result = super(ProjectPermission, self) \
+            .has_object_permission(request, view, project.organization)
+
+        if not result:
+            return result
+
+        if not request.user.is_authenticated():
```

Comment:
Do organization API keys need access to attachments?

Reply:
This points me to a pretty serious bug right now, thanks! 
To your question: Are these the keys for integrations or is that yet another thing?

Reply:
I was thinking of the older organization API tokens, so it likely isn't that relevant.

Reply:
Thanks :) Those seem to be deprecated, so I guess we can ignore that 👍